stripe-node/api/charge.js

```diff
@@ -16,7 +16,7 @@ const sendData = (req, res, next) => {
   };
   let amount = parseInt(req.body.amount) / 100;
   if(amount < 5) {
-    res.send("The minimum contribution amount is $5.00");
+    res.status(400).send("The minimum contribution amount is $5.00");
     return;
   }
```

Comment:
This is the server-side test for min amount. We could probably go further in our tests here.